Parallel builds broken

The changes in libpagekite/Makefile.am since v0.91.160811 break parallel builds:

```
arm-v4t-linux-gnueabi-gcc -DHAVE_CONFIG_H -I. -I..     -I../include -g -O2 -MT tests-tests.o -MD -MP -MF .deps/tests-tests.Tpo -c -o tests-tests.o `test -f 'tests.c' || echo './'`tests.c
make[3]: *** No rule to make target `../libpagekite/libpagekite.la', needed by `tests'.  Stop.
make[3]: *** Waiting for unfinished jobs....
```

--- a/libpagekite/Makefile.am
+++ b/libpagekite/Makefile.am
@@ -10,7 +10,7 @@ libpagekite_la_CFLAGS = $(LIBEV_CFLAGS)
 libpagekite_la_LDFLAGS = -lpthread $(OPENSSL_LIBS) $(LIBEV_LIBS) \
         -lm -no-undefined -version-info 1:0:0
 
-tests_LDADD = $(top_srcdir)/libpagekite/libpagekite.la
+tests_LDADD = libpagekite.la
 tests_CFLAGS = $(LIBEV_CFLAGS) -I$(top_srcdir)/include
 
 if HAVE_JAVA

